drivers/watchdog/wdt_iwdg_stm32.c

```diff
@@ -52,6 +52,10 @@
 #define IWDG_SR_UPDATE_TIMEOUT	(6U * IWDG_PRESCALER_MAX * \
 				 MSEC_PER_SEC / LSI_VALUE)
 
+/* Store the calculated values to write in the iwdg registers */
+uint32_t prescaler;
+uint32_t reload;
+
```

Comment:
Even though we only have a single IWDG instance, I think these should be stored in the `struct iwdg_stm32_data`.

Reply:
yes that's still possible